### PR TITLE
Update rust.mdx

### PR DESCRIPTION
**Pull request by Thoralf-M**
## Type of change

- Documentation Fix

### CODE REVIEW

File: docs/build/iota-sdk/1.0.0/docs/getting-started/rust.mdx
Comment (Dr-Electron) [inline docs/build/iota-sdk/1.0.0/docs/getting-started/rust.mdx:1]: Can't we import that code from somewhere?

Comment (Thoralf-M) [reply docs/build/iota-sdk/1.0.0/docs/getting-started/rust.mdx:1]: Currently we don't have examples without env vars like this, opened an issues for that https://github.com/iotaledger/iota-wiki/issues/1139 https://github.com/iotaledger/iota-sdk/issues/1115

Comment (Dr-Electron) [reply docs/build/iota-sdk/1.0.0/docs/getting-started/rust.mdx:1]: Got it. Don't forget updating the wiki once that is done ;)

**Review by Dr-Electron — APPROVED** [review]

File: docs/build/iota-sdk/1.0.0/docs/getting-started/rust.mdx
Comment (Thoralf-M) [reply docs/build/iota-sdk/1.0.0/docs/getting-started/rust.mdx:1]: There is a wiki issue so it's not forgotten :trollface:

**Pull request merged**